serverTimestamp.properties

```diff
@@ -0,0 +1,7 @@
+comodaca=http://timestamp.comodoca.com/authenticode
+isectigo=http://timestamp.sectigo.com
+comodocarfc31611=http://timestamp.comodoca.com/rfc3161
```

Comment:
Is the final "1" a typo? Should it be "comodocarfc3161"?

Reply:
Good find! Fixed :-)